src/Templater.js

```diff
@@ -20,13 +20,13 @@ export default class Templater {
 
         this._built = false;
 
-        this._i18n = config.i18n || this._default_i18n();
+        this._i18n = config.i18n || this._mapper();
```

Comment:
How about this variable itself? I guess we need to retain i18n for backward compatibility. but the internal variable can be renamed?

this._mapper = config.mapper || config.i18n || this._mapper();

Reply:
I see..

Reply:
fixed